nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,77 @@
+const express = require("express");
+const app = express();
+app.use(express.urlencoded({extended: true}))
+
+app.get("/", async(req, res) => res.send("nodejs week3 homework"));
+
+
+app.get("/add", (req, res) =>{
+    let values = Object.values(req.query)
+    values = values.flat()
+    values = values.map(n=> Number(n))
+    let sum = values.reduce((a,b) => a + b)
```

Comment:
```suggestion
    const values = Object.values(req.query).flat();
    const sum = values.reduce((a, b) => Number(a) + Number(b));
```
    

I think it can be written simpler :) 
Same for all operations: subtract, multiply and division